ui/main.py

```diff
@@ -65,6 +65,11 @@ def run(driver, config):
     store.dispatch(actions.trigger())
     button_loop(driver)
 
+    state    = store.get_state()
```

Comment:
Can you make this a seperate `def`please. Also there is some extra space in front of the equals.